project-p1-first-webpage-master/code/style.css

```diff
@@ -0,0 +1,183 @@
+/* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
+
+body {
+  background-color:#faf3dd;
+  margin: 20px 60px 20px 60px;
+  font-family: 'Architects Daughter', cursive;
+}
+
+.center {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 400px;
+  }
+
+header {
+  background-image: url("Images/Page.JPG");
+  background-size: cover; 
+  object-position: bottom;
+  border-color: grey; 
+  border-style: double;
+  text-align: center;
+  height: 700px;
+  padding: 100px; 
+}
+
+h1 {
+  font-family: 'Charmonman', cursive;
+  font-family: 'Eagle Lake', cursive;
+  font-family: 'Sunshiney', cursive;
```

Comment:
A suggestion here and on line 37-39 is to put the font-families on one row, starting with the one that you want to use the most, for example:
`font-family: 'Charmonman', 'Eagle Lake', 'Sunshiney', cursive;`